src/Components/AstroChat/useAstro.ts

```diff
@@ -193,7 +207,8 @@ export const useAstro = (messageProcessors: Array<MessageProcessor>, astroOption
           );
         }
 
-        const postTalkResponse = postTalk(message, buildMetadata());
+        const postTalkResponse = (await postTalk(message, buildMetadata())).response;
+        console.log('postTalkResponse: ', postTalkResponse);
 
```

Comment:
I would think most of the changes would live around here to translate between the new and old types.